src/pushsource/_impl/model/ami.py

```diff
@@ -99,6 +99,21 @@ class AmiPushItem(VMIPushItem):
     )
     """Billing codes associated with this image."""
 
+    description = attr.ib(type=str, default=None, validator=optional_str)
+    """A description for the image"""
```

Comment:
Sorry, I missed this one.

I believe we can remove it as well and use the [description](https://github.com/release-engineering/pushsource/blob/master/src/pushsource/_impl/model/vms.py#L57-L58) from the super class `VMIPushItem`.

@jajreidy WDYT?

Reply:
Ah yeah that makes sense.